indra/tests/test_statements_serialization.py

```diff
@@ -1,88 +1,99 @@
 from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
```

Comment:
I tend to prefer to keep these as boilerplate imports because this way if the code is ever extended to call str() in Python 2 (e.g., to convert an int to a string) it will return a future.newstr, which is unicode compatible, rather than a Python 2 bytestring.